docs/source/split.rst

```diff
@@ -119,7 +119,7 @@ The ``findwhen`` method test a condition and returns a vector of ``Date`` or ``D
     julia> green = findwhen(ohlc["Close"] .> ohlc["Open"]);
 
     julia> typeof(green)
-    Array{Date,1}
+    Vector{Date}
```

Comment:
This isn't correct. The type system doesn't know what a `Vector` is; it's just a type alias. Regular `Array`s always print as `Array{T,N}`.